ts/queue.ts

```diff
@@ -38,7 +38,7 @@ export default class Queue {
       expires,
       deadLetterExchange,
       deadLetterRoutingKey,
-      maxLength
+      maxLength,
```

Comment:
Nit: You could destructure arguments here

Reply:
Hello @nikostoulas 
made a change, plz have a look.